1st-page/Blog.html

```diff
@@ -0,0 +1,164 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Page3 Lesson1 Assignment</title>
+</head>
+
+<style>
```

Comment:
I would advise you move these stylesheets over to a dedicated css file applied to all pages.